pkg/capabilities/datastreams/types.go

```diff
@@ -73,6 +73,12 @@ type SignersMetadata struct {
 	MinRequiredSignatures int
 }
 
+type StreamsTriggerPayload struct {
+	Payload   []FeedReport
+	Metadata  SignersMetadata
+	Timestamp int64
+}
+
 type ReportCodec interface {
 	// unwrap reports and convert to a list of FeedReport
 	Unwrap(wrapped values.Value) ([]FeedReport, error)
```

Comment:
This threw me off initially when reading the rest of the code. You are silently changing the expectation on the Codec to Unwrap the whole Event, instead of just the payload. This is OK but I wonder if we can document it somehow. Maybe change the "wrapped" param name to "wrappedTriggerEvent"? Or technically "wrappedTriggerEventOutputs", which doesn't sound great :/

Reply:
I'll add some comments